tests/views/test_load_sample.py

```diff
@@ -30,6 +32,10 @@ def test_upload_social_sample_file(client):
                                data={'sample': (BytesIO(b'header\nline'), 'sample.csv')})
 
     load_sample_patch.assert_called_once_with(ANY, collection_exercise_id='4c27fcbf-e069-44db-a239-99c9cd70a1d0',
-                                              action_plan_id='4c27fcbf-e069-44db-a239-99c9cd70a1d0')
+                                              action_plan_id='4c27fcbf-e069-44db-a239-99c9cd70a1d0',
+                                              host=Config.RABBITMQ_HOST, port=Config.RABBITMQ_PORT,
+                                              vhost=Config.RABBITMQ_VHOST, exchange=Config.RABBITMQ_EXCHANGE,
+                                              user=Config.RABBITMQ_USER, password=Config.RABBITMQ_PASSWORD,
+                                              queue_name=Config.RABBITMQ_QUEUE)
```

Comment:
Use the app's config object here instead? Similar to [this](https://github.com/ONSdigital/census-rm-ops/blob/master/app/controllers/action_controller.py#L2-L9)